Environmental/DRAXIS/.env.example

```diff
@@ -0,0 +1,6 @@
+KAFKA_HOST=kafka_host
+KAFKA_PORT=kafka_port
+ELASTICSEARCH_HOST=elasticsearch_host
+ELASTICSEARCH_PORT=elasticsearch_port
+WEATHER_FORECAST_DRAXIS_API=weather_forecast_draxis_api_url
```

Comment:
if this is the url, then maybe it is better for this to be `DRAXIS_API_WEATHER_FORECAST_URL`